src/app/catalogue/asset/page.js

```diff
@@ -0,0 +1,8 @@
+import DatasetInfoComponent from './components/DatasetInfoComponent'
+export default function Page () {
+  return (
+    <div className='bg-sedimark-light-blue pt-2'>
+      <DatasetInfoComponent />
```

Comment:
Question: There's only this component as a child because you foresee that the recommended offerings list will be another children ?

Reply:
Yes exactly, I'm trying to componentize everything separately, I might change more stuff through the development